x-pack/plugins/cloud_security_posture/public/components/csp_counter_card.tsx

```diff
@@ -44,6 +44,9 @@ export const CspCounterCard = (counter: CspCounterCardProps) => {
           display: 'flex',
           flexDirection: 'column',
           justifyContent: 'space-around',
+          '.euiText h6': {
+            textTransform: 'capitalize',
+          },
```

Comment:
in the EUI page for this stat component it looks like the texts are not transformed by the EuiStat component itself, which makes me think that we should be able to solve that without using CSS classes which were not exposed directly.

maybe replace the `descriptionElement="h6"` with `descriptionElement="strong"`? or pass a node instead of just text?